src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -233,6 +234,20 @@ class Pex:
     digest: Digest
     name: str
     python: PythonExecutable | None
+    pex_path: Tuple[Pex, ...]
+
+    @classmethod
+    def pex_path_closure(cls, pexes: Iterable[Pex]) -> OrderedSet[Pex]:
```

Comment:
Thoughts on making this a top-level function? Imo it reads nicer and fits with the project's style better to be `pex_path_closure` rather than `Pex.pex_path_closure`. The extra namespacing doesn't add much.

Reply:
Done.